### PR TITLE
add support of sbom for JS

### PR DESCRIPTION
**Pull request by eyalatox**

### CODE REVIEW

File: packages/backend/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "types": "./src/index.ts",
   "main": "./dist/index.js",
   "scripts": {
-    "start": "nodemon -r dotenv/config ./src/index.ts dotenv_config_path=../../.env dotenv_config_debug=true",
+    "start": "ts-node -r dotenv/config ./src/index.ts dotenv_config_path=../../.env dotenv_config_debug=true",
```
Comment (nvuillam) [inline packages/backend/package.json:8]: @eyalatox shouldn't you add ts-node in devDependencies , no we can be sure than it is available after `npm install` ?

**Review by nvuillam — CHANGES_REQUESTED** [review]
Plz check comment + update CHANGELOG.md :)

File: packages/backend/package.json (hunk 1 above)
Comment (eyalatox) [reply packages/backend/package.json:8]: right, I'll add it

**Review by nvuillam — APPROVED** [review]
:)

File: packages/backend/package.json (hunk 1 above)
Comment (itayox) [reply packages/backend/package.json:8]: ts-node isn't doing watch... why did you replace `nodemon`?

**Pull request merged**